### ISSUE TITLE
Tablet area confines to windows which grab input

### ISSUE THREAD

**Issue by Stoppedpuma** [hyprwm/Hyprland#7481]
### Regression?

Yes

### System Info and Version

<details> <summary>System/Version info</summary>

```sh
Hyprland, built from branch main at commit f634b9e61af7dae0ca70379bd207eea45bb417d1  (Fix crash reports having execute permission).
Date: Thu Aug 22 14:28:01 2024
Tag: v0.42.0-58-gf634b9e6, commits: 5127
```

</details>

### Description

This issue is possibly related to #6023 and has been happening for months. The input from tablets will be confined to applications which grab input such as games.

### How to reproduce

Change resolution on an application which grabs input while using a tablet.

### Crash reports, logs, images, videos

Example using the OpenTabletDriver debug visualiser with osu!. This example shows making the same square shape and it becoming taller vertically, becoming closer to a vertical rectangle when being tiled instead of the same square when in fullscreen.

[tablet.webm](https://github.com/user-attachments/assets/b8b21926-0624-48be-a449-77b928b47141)

**Comment by vaxerski** [comment]
that's expected behavior no?

**Comment by Stoppedpuma** [comment]
No. Tablets should be mapped using outputs, not applications. The only case it should be mapped to applications is if the application itself has a driver or something built in and is being used.

**Comment by Agent00Ming** [comment]
Pretty sure osu-lazer has it's own way of getting tablet input as it gets the tablet input (movement) even when the window isn't focused. Otherwise, OTD in absolute mode presents itself as a normal mouse (virtual mouse device) which gets grabbed by pointer constraints.

**Comment by Stoppedpuma** [comment]
osu! has OTD built in and will grab tablet input if it's enabled, otherwise it will just use the systems driver. This setting is not enabled on my end and input is handled correctly in other Wayland compositors such as Sway.

**Comment by vaxerski** [comment]
Hello there!

This issue has been closed, as we are moving from issues to discussions. More about this can be found [here](https://wiki.hyprland.org/Contributing-and-Debugging/Issue-Guidelines/) and in #9854.

Firstly, please make sure this issue is still relevant in latest Hyprland. If so, we ask you to open a [discussion](https://github.com/hyprwm/Hyprland/discussions) (please read the discussion guidelines first, in the pinned post)

Thank you for your understanding!
